diff -ur babl-0.0.14-o/babl/Makefile.am babl-0.0.14/babl/Makefile.am
--- babl-0.0.14-o/babl/Makefile.am	2007-02-14 16:21:20.000000000 -0700
+++ babl-0.0.14/babl/Makefile.am	2007-06-10 17:18:09.000000000 -0600
@@ -51,8 +51,11 @@
 libbabl_@BABL_API_VERSION@_la_SOURCES= $(h_sources) $(c_sources)
 libbabl_@BABL_API_VERSION@_la_LIBADD=\
 	base/libbase.la \
+	$(DL_LIB) $(MATH_LIB) \
 	@LTLIBOBJS@
 
+EXTRA_LTLIBRARIES =
+
 libbabl_@BABL_API_VERSION@_la_LDFLAGS= \
 	${no_undefined}                \
 	-version-info $(BABL_LIBRARY_VERSION)
Only in babl-0.0.14/babl: Makefile.am~
